src/sonic-config-engine/tests/test_j2files.py

```diff
@@ -293,7 +294,7 @@ def test_l2switch_template_dualtor(self):
     def test_qos_arista7050_render_template(self):
         self._test_qos_render_template('arista', 'x86_64-arista_7050_qx32s', 'Arista-7050-QX-32S', 'sample-arista-7050-t0-minigraph.xml', 'qos-arista7050.json')
 
-    def do_test_qos_and_buffer_arista7800r3_48cq2_lc_render_template(self, platform, hwsku):
+    def do_test_qos_and_buffer_arista7800r3_48cq2_lc_render_template(self, platform, hwsku, minigraph, qos_sample_output, buffer_sample_output):
```

Comment:
Why do we need this function? Can we not reuse the function "do_test_qos_and_buffer_lc_render_template" which is used by other chassis hwskus?

Reply:
@neethajohn  Since other SKUs has multiple asics and these SKU have single asic, there is difference in location of config files in both cases. Hence using different functions.

Reply:
The only diff is in L334. You can extend the function arguments to include a multi_asic flag with a default value. If that is true, add the path to asic0

Reply:
Commited the feeback.